tables/Tables/NullTable.cc

```diff
@@ -252,9 +252,11 @@ Bool NullTable::adjustRownrs (rownr_t, Vector<rownr_t>&,
 }
 
 BaseTable* NullTable::doSort (PtrBlock<BaseColumn*>&,
-			      const Block<CountedPtr<BaseCompare> >&,
-			      const Block<Int>&,
-			      int)
+                             const Block<CountedPtr<BaseCompare> >&,
+                             const Block<Int>&,
+                             int,
+                             std::shared_ptr<Vector<rownr_t>>,
+                             std::shared_ptr<Vector<size_t>>)
```

Comment:
For the future (not for now) it might be a good idea to start using the `override` keyword. It might find some issues. It's possible to enable warnings on not using it with `-Wsuggest-override`.